begin: 1666783789.81669
cmd: /opt/fio/bin/fio --name="io_uring-xnvme" --rw="randread" --size="12G" --iodepth="8"
  --bs="512" --direct="1" --thread="1" --time_based="1" --runtime="7" --ramp_time="3"
  --norandommap="1" --allow_file_create="0" --ioengine="external:/usr/local/lib/x86_64-linux-gnu/libxnvme-fio-engine.so"
  --xnvme_be="linux" --xnvme_async="io_uring" --xnvme_sync="nvme" --xnvme_admin="nvme"
  --xnvme_dev_nsid="1" --filename="/dev/nvme0n1" --output-format="normal,json" --output="/tmp/fio-output_512_8_io_uring-xnvme_2.txt"
  --cpus_allowed="1"
cwd: None
elapsed: 11.817346572875977
end: 1666783801.6340365
err: 0
output_dpath: /home/safl/git/spdkvf2022/cijoe-output/003_fio
output_fpath: /home/safl/git/spdkvf2022/cijoe-output/003_fio/cmd_359.output
